receiptui/src/main/java/com/hyperwallet/android/ui/receipt/view/ReceiptDetailActivity.java

```diff
@@ -84,10 +84,6 @@ public boolean onSupportNavigateUp() {
 
     @Override
     public void onBackPressed() {
-        getWindow().clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
-        getWindow().addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
-        getWindow().setStatusBarColor(ContextCompat.getColor(this, R.color.colorPrimaryDark));
-        getWindow().getDecorView().setSystemUiVisibility(0);
```

Comment:
whole implementation is wrong IMO. You are not fixing the root cause. 
In App we override theme colors as required. You can see `R.color.colorPrimaryDark` is overriden in App as
`<color name="colorPrimaryDark">#994625</color>`
See `#994625`

Reply:
fixed in app, I am closing this PR